app/src/debug/java/org/simple/clinic/DebugClinicApp.kt

```diff
@@ -89,7 +89,7 @@ class DebugClinicApp : ClinicApp() {
         .appModule(AppModule(this))
         .loginModule(object : LoginModule() {
           override fun appLockConfig(): Single<AppLockConfig> {
-            return Single.just(AppLockConfig(lockAfterTimeMillis = TimeUnit.SECONDS.toMillis(4)))
+            return Single.just(AppLockConfig(lockAfterTimeMillis = TimeUnit.MINUTES.toMillis(5)))
```

Comment:
Let's retain 4s because it's easier to test the lockscreen during development?

Reply:
Oh, this was an accidental  change. I'll revert it.

Reply:
Done.